pkg/network/tracer/tracer.go

```diff
@@ -182,7 +182,13 @@ func NewTracer(config *config.Config) (*Tracer, error) {
 		}
 		defer offsetBuf.Close()
 
-		mgrOptions.ConstantEditors, err = runOffsetGuessing(config, offsetBuf)
+		// Offset guessing has been flaky for some customers, so if it fails we'll retry it up to 5 times
+		for i := 0; i < 5; i++ {
+			mgrOptions.ConstantEditors, err = runOffsetGuessing(config, offsetBuf)
+			if err == nil {
```

Comment:
I wonder if it would make sense to do a `sleep` between retries.

Reply:
🤔  I don't think it could hurt, I'll add that!